This fix compilation errors on android-arm and android-x86
```
  /home/runner/work/boinc/boinc/3rdParty/android/vcpkg/buildtrees/usockets/src/v0.8.8-af4482ba56.clean/src/eventing/epoll_kqueue.c:104:30: error: use of undeclared identifier 'EPOLL_CLOEXEC'
      loop->fd = epoll_create1(EPOLL_CLOEXEC);
```

--- a/ports/usockets/CMakeLists.txt
+++ b/ports/usockets/CMakeLists.txt
@@ -27,14 +27,16 @@ else()
     target_compile_definitions(uSockets PRIVATE -DLIBUS_NO_SSL)
 endif()
 
-if(WIN32)
+if(WIN32 OR ANDROID)
     # https://github.com/uNetworking/uSockets/blob/8606de6414a102c55bef8e8ef3391932d7e8df6a/src/libusockets.h#L339-L348
     find_package(libuv CONFIG REQUIRED)
     target_link_libraries(uSockets PRIVATE $<IF:$<TARGET_EXISTS:libuv::uv_a>,libuv::uv_a,libuv::uv>)
     target_compile_definitions(uSockets PRIVATE -DLIBUS_USE_LIBUV)
 
-    # https://github.com/uNetworking/uSockets/blob/8606de6414a102c55bef8e8ef3391932d7e8df6a/src/libusockets.h#L35
-    target_link_libraries(uSockets PRIVATE ws2_32)
+    if(WIN32)
+        # https://github.com/uNetworking/uSockets/blob/8606de6414a102c55bef8e8ef3391932d7e8df6a/src/libusockets.h#L35
+        target_link_libraries(uSockets PRIVATE ws2_32)
+    endif()
 endif()
 
 

--- a/ports/usockets/unofficial-usockets-config.cmake
+++ b/ports/usockets/unofficial-usockets-config.cmake
@@ -1,5 +1,5 @@
 include(CMakeFindDependencyMacro)
-if(WIN32)
+if(WIN32 OR ANDROID)
     find_dependency(libuv CONFIG)
 endif()
 if("@WITH_OPENSSL@")

--- a/ports/usockets/vcpkg.json
+++ b/ports/usockets/vcpkg.json
@@ -1,14 +1,14 @@
 {
   "name": "usockets",
   "version": "0.8.8",
-  "port-version": 2,
+  "port-version": 3,
   "description": "Miniscule cross-platform eventing, networking & crypto for async applications",
   "homepage": "https://github.com/uNetworking/uSockets",
   "license": "Apache-2.0",
   "dependencies": [
     {
       "name": "libuv",
-      "platform": "windows"
+      "platform": "windows | android"
     },
     {
       "name": "vcpkg-cmake",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -9810,7 +9810,7 @@
     },
     "usockets": {
       "baseline": "0.8.8",
-      "port-version": 2
+      "port-version": 3
     },
     "usrsctp": {
       "baseline": "0.9.5.0",

--- a/versions/u-/usockets.json
+++ b/versions/u-/usockets.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "080448b6175d620e1b986327d0a10caee41946d4",
+      "version": "0.8.8",
+      "port-version": 3
+    },
+    {
       "git-tree": "28a78040653744fdea484a20a0cfee8b14f14312",
       "version": "0.8.8",
       "port-version": 2

